core/utils/xml.ts

```diff
@@ -14,40 +14,69 @@
 import * as goog from '../../closure/goog/goog.js';
 goog.declareModuleId('Blockly.utils.xml');
 
+import * as deprecation from './deprecation.js';
+
 
 /**
- * Namespace for Blockly's XML.
+ * Injected dependencies.  By default these are just (and have the
+ * same types as) the corresponding DOM Window properties, but the
+ * Node.js wrapper for Blockly (see scripts/package/node/core.js)
+ * calls injectDependencies to supply implementations from the jsdom
+ * package instead.
+ */
+let {document, DOMParser, XMLSerializer} = globalThis;
+
+/**
+ * Inject dependencies.  Used by the Node.js wrapper for Blockly (see
+ * scripts/package/node/core.js) to supply implementations from the
+ * jsdom package instead.  While they may be set individually, it is
+ * normally the case that all three should be supplied and sourced
+ * from the same JSDOM instance.
  *
- * @alias Blockly.utils.xml.NAME_SPACE
+ * @param dependencies Options object contianing dependencies to set.
  */
-export const NAME_SPACE = 'https://developers.google.com/blockly/xml';
+export function injectDependencies(dependencies: {
+  document?: Document,
+  DOMParser?: typeof DOMParser,
+  XMLSerializer?: typeof XMLSerializer,
+}) {
+  ({
```

Comment:
Yes, because `{ /* ... */ } =` at the start of a statement gets parsed as a block followed by an (illegal) `=` token, rather than as a destructuring assignment.